test/voting_to_change_keys_test.js

```diff
@@ -290,6 +290,29 @@ contract('Voting to change keys [all features]', function (accounts) {
         {from: votingKey}
       ).should.be.rejectedWith(ERROR_MSG)
     })
+    it('cannot create a ballot with earlier removed mining key', async () => {
```

Comment:
Let's also add a test for `createBallot` which would try to swap to earlier removed miningKey.